polar_route/route_planner/crossing_smoothing.py

```diff
@@ -99,7 +97,6 @@ def rhumb_traveltime_in_cell(cellbox, cp, sp, s, u, v):
         θ = y/(2*6371*1000) + λ
         C1 = s**2 - u**2 - v**2
         z = x*np.cos(θ)
-        r1 = np.cos(λ) / np.cos(θ)
```

Comment:
Although these parameters are not used the equations need to remain to be consistent with the mathematics.